lib/node_modules/@stdlib/array/fixed-endian-factory/README.md

```diff
@@ -607,6 +607,39 @@ var idx = arr.lastIndexOf( 5.0 );
 // returns -1
 ```
 
+<a name="method-toSorted"></a>
```

Comment:
This method should be inserted into the docs such that methods are listed in alphabetical order.